oiio-sys/build.rs

```diff
@@ -17,12 +17,21 @@ fn main() -> Result<()> {
         return Ok(());
     }
 
-    let pkgconfig = pkg_config::probe_library("OpenImageIO")?;
+    let pkgconfig_ooio = pkg_config::probe_library("OpenImageIO")?;
+    let pkgconfig_imath = pkg_config::probe_library("Imath")?;
+    let pkgconfig_gdal = pkg_config::probe_library("gdal")?;
```

Comment:
What's depending on gdal?

Reply:
I'm new to Rust and still push buttons until it works. It looks like it was the code I tried to compile unrelated to oiio-bind that required gdal, and adding it here actually fixed the linking issue. But it should be in my project's repository.

I can't seem to reproduce my issues by running `cargo build` in oiio-bind itself. Maybe the best way to continue will be to add some test cases that uses the library. This will point out what is required instead of pulling in random pkg-config calls.

I believe that would be the best way and this PR should probably be closed. Maybe I'll give it a shot.